iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandler.java

```diff
@@ -330,6 +333,15 @@ public boolean canProvideBasicStatistics() {
     return true;
   }
 
+  @Override
+  public StorageFormatDescriptor supportsLoadData(org.apache.hadoop.hive.metastore.api.Table tbl) {
+    if (tbl.getParameters() != null) {
+      String format = tbl.getParameters().get("write.format.default");
```

Comment:
do we have constants for the tbl properties?

Reply:
done